node/sc/bridge_manager.go

```diff
@@ -39,7 +39,7 @@ import (
 const (
 	TokenEventChanSize  = 10000
 	BridgeAddrJournal   = "bridge_addrs.rlp"
-	maxPendingNonceDiff = 20000 // TODO-Klaytn-ServiceChain: update this limitation. Currently, 2 * 10000 TPS.
+	maxPendingNonceDiff = 1000 // TODO-Klaytn-ServiceChain: update this limitation. Currently, 2 * 500 TPS.
```

Comment:
I think 20,000 is not big to handle the pending nonce. Is there any reason to decrease this value?

Reply:
From our performance test, our system can process 500 value transfer.
So I think this value is too much. And also this value is the limitation for each one way bridge pair.
If there are many bridge, this value is quite much. I think.
Anyway, I will tune this value later.